### PR TITLE
dev_start.sh: retire unused yolo3d volume

### PR DESCRIPTION
**Pull request by storypku**

### CODE REVIEW

File: docker/scripts/dev_start.sh [hunk 1]
```diff
@@ -439,14 +439,8 @@ function mount_map_volumes() {
...
     local volume_conf=
-    if [ "${FAST_MODE}" = "no" ]; then
-        # YOLO3D
-        local yolo3d_volume="apollo_yolo3d_volume_${USER}"
-        local yolo3d_image="${DOCKER_REPO}:yolo3d_volume-${TARGET_ARCH}-latest"
-        docker_restart_volume "${yolo3d_volume}" "${yolo3d_image}"
-        volume_conf="${volume_conf} --volumes-from ${yolo3d_volume}"
-    fi
-
+    # if [[ "${FAST_MODE}" == "no" ]]; then
```
Comment (Capri2014) [inline docker/scripts/dev_start.sh:442]: why still leave these commented lines?

Comment (storypku) [reply docker/scripts/dev_start.sh:442]: Intended for placeholder for other non-fast-mode volumes. Although It seems that nearly every volume is in-fast-mode.

Comment (storypku) [reply docker/scripts/dev_start.sh]: > why still leave these commented lines?

Cleaned up.

File: docker/scripts/dev_start.sh [hunk 2]
```diff
@@ -425,7 +425,7 @@ function mount_map_volumes() {
         done
     fi
 
-    if [ "$FAST_MODE" = "no" ]; then
+    if [[ "$FAST_MODE" == "no" ]]; then
         for map_name in ${DEFAULT_MAPS[@]}; do
```
Comment (storypku) [inline docker/scripts/dev_start.sh:429]: As per google shell style guide, `[[  ]]` is preferred.

**Review by changsh726 — APPROVED** [review]

**Review by changsh726 — APPROVED** [review]

**Pull request merged**